### ISSUE TITLE
IDE Javascript auto-completion "lag"

### ISSUE THREAD

**Issue by phidoi247** [PhaserEditor2D/PhaserEditor#90]
Dear PhaserEditor,

After test, the UTF8 issue of the IDE has been fixed. Thank you for your hard working.

But there are some problems still not fixed. I'm working in a project with about 30 .canvas files. The IDE will be "lag" if i use the "ctrl + space" or copy some code line. So i just use the IDE for design and use Visual Studio Code for coding. Can you fix the "lag" issue of the IDE ?

**Comment by PyvesB** [comment]
[WildWebDeveloper](https://github.com/eclipse/wildwebdeveloper) is also another good alternative for JavaScript and TypeScript edition in Eclipse. :wink:

**Comment by PhaserEditor2D** [comment]
> [WildWebDeveloper](https://github.com/eclipse/wildwebdeveloper) is also another good alternative for JavaScript and TypeScript edition in Eclipse.

Yes, and it is the default option in v2 of Phaser Editor :)

The v1 series of the editor is attached to an old Eclipse version, I don't know if it is compatible with WWD.

**Comment by phidoi247** [comment]
> > [WildWebDeveloper](https://github.com/eclipse/wildwebdeveloper) is also another good alternative for JavaScript and TypeScript edition in Eclipse.
> 
> Yes, and it is the default option in v2 of Phaser Editor :)
> 
> The v1 series of the editor is attached to an old Eclipse version, I don't know if it is compatible with WWD.

do you still update for v1 ide?

**Comment by PhaserEditor2D** [comment]
> do you still update for v1 ide?

Yes, but only with bug fixes and minor features. or blocking issues if you are a paid user. We are not going to update it to a newer Eclipse version. The main effort is to move forward v2.
